general-concepts/package-maintainers/text.xml

```diff
@@ -5,6 +5,63 @@
 
 <body>
 
+<section>
+<title>Maintainers</title>
+<body>
+
+<p>
+Package maintainers are responsible for ebuild maintenance tasks —
+bumping them to new upstream releases, updating as necessary wrt
+policies, replying to bugs, etc. They are listed in package's
+<uri link="::ebuild-writing/misc-files/metadata/">metadata.xml</uri>
+file. When assigning bugs, the first maintainer listed becomes the bug's
+assignee and the remaining maintainers are added to CC.
+</p>
+
+<p>
+Package maintainers have authority over their maintained packages.
+Unless specified otherwise, you should request maintainers' approval
+before committing to their packages. The exceptions to this rule are
+trivial changes implied by your own actions, e.g. dependency updates
+as a result of package moves.
+</p>
+
+<p>
+In case of maintainer's inactivity, a customary tool of maintainer
+timeout may be used. This timeout can be invoked if none of
+the package's maintainers reply to a patch within 4 weeks of it being
+attached or linked to a correctly assigned bug, or being sent
+to the maintainer.
+</p>
+
+<p>
+All packages newly added to the Gentoo repository must have at least
+one maintainer specified. New maintainers can only be added with their
+consent. In particular, it is not acceptable to add generic projects
+(such as the Python project) as package maintainers without approval
+from their members or against their explicit policy.
+</p>
+
+<p>
+Maintainers do not necessarily have to have commit access. Maintainers
+without it are called proxied maintainers, and they proxy their commits
+via another developer or project. All packages maintained by proxied
```

Comment:
Proxied maintainers don't proxy; proxy maintainers do. :-)

This should probably say something like "... and another developer or project pushes their commits as a proxy."